dag/stage/aneel/task_factory.py

```
@@ -0,0 +1,104 @@
+from typing import Callable as _Callable
```

Ruff (0.12.2)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)